### DISCUSSION TITLE
paddle中的en_dict.txt进行修改是否可以提高准确性？

### DISCUSSION THREAD

**Discussion by happysunguy841031** [PaddlePaddle/PaddleOCR#13713] [Q&A] [answered]
一般我们在运用的时候会根据实际场景设定语言，比如ch或en，如果能确定识别的内容只会出现A-Z(大写)及0-9(数字)，是否可以修改字典，即 paddle 在 OCR 所使用的 en_dict.txt 删除小写及符号？另外想请问为什么有的电脑打开是乱码，有的不是？就算直接输入A-Z0-9还是无法正常识别？谢谢

**Comment by GreatV** [comment, marked as answer]
修改了字典你得重新训练才行

**Reply by happysunguy841031** [reply]
> 修改了字典你得重新训练才行

谢谢您的回复。所以我是不是需要

【第一步】使用 PPOCRLabel 标记完成，导出文件
【第二步】修改字典
【第三步】修改标记+识别训练指向的字典
【第四步】再开始训练

再次感谢！

**Reply by GreatV** [reply]
是的
